src/main/kotlin/org/acejump/search/Tagger.kt

```diff
@@ -37,6 +37,10 @@ internal class Tagger(private val editor: Editor) {
     tagMap = HashBiMap.create()
   }
 
+  fun getMarkByOffset(offset: Int): String? {
```

Comment:
Since we already have something called a [`Mark`](https://github.com/acejump/AceJump/blob/2c0a236ed84df23b5ab353657fe251157b567d9f/src/main/kotlin/org/acejump/search/TaggingResult.kt#L7), to avoid confusion maybe let's call this `getTagByOffset` instead?